docs/ops.html

```diff
@@ -586,7 +582,7 @@
   socket.request.max.bytes=104857600
   socket.receive.buffer.bytes=1048576
   socket.send.buffer.bytes=1048576
-  queued.max.requests=16
+  queued.max.requests=500
```

Comment:
@granthenke : Hmm, how many brokers are there? Note that num.replica.fetchers is per source broker. So, if you have a 10 node broker, each broker is already fetching from 9 other brokers concurrently even with 1 num.replica.fetchers. The down side of more num.replica.fetchers is that it will increase the request rate on the broker.